ai-data/managed-inference/quickstart.mdx

```diff
@@ -1,38 +1,38 @@
 ---
 meta:
-  title: LLM Inference - Quickstart
-  description: Start with Scaleway LLM Inference for secure, scalable AI model deployment in Europe's premier platform. Privacy-focused, fully managed.
+  title: Managed Inference - Quickstart
+  description: Start with Scaleway Managed Inference for secure, scalable AI model deployment in Europe's premier platform. Privacy-focused, fully managed.
 content:
-  h1: LLM Inference - Quickstart
-  paragraph: This page explains all the concepts related to LLM Inference
+  h1: Managed Inference - Quickstart
+  paragraph: This page explains all the concepts related to Managed Inference
 tags:
 categories:
   - ai-data
 ---
 
-Scaleway LLM Inference is the first European LLM Inference platform on the market. It is a scalable and secure inference engine for Large Language Models (LLMs).
+Scaleway Managed Inference is the first European Managed Inference platform on the market. It is a scalable and secure inference engine for Large Language Models (LLMs).
 
-Scaleway LLM Inference is a fully managed service that allows you to serve generative AI models in a production environment.
-With Scaleway LLM Inference, you can easily deploy, manage, and scale LLMs without worrying about the underlying infrastructure.
+Scaleway Managed Inference is a fully managed service that allows you to serve generative AI models in a production environment.
+With Scaleway Managed Inference, you can easily deploy, manage, and scale LLMs without worrying about the underlying infrastructure.
 
-Here are some of the key features of Scaleway LLM Inference:
+Here are some of the key features of Scaleway Managed Inference:
 
-* **Easy deployment**: Deploy state-of-the-art open weights LLMs with just a few clicks. Scaleway LLM Inference provides a simple and intuitive interface for generating dedicated endpoints.
+* **Easy deployment**: Deploy state-of-the-art open weights LLMs with just a few clicks. Scaleway Managed Inference provides a simple and intuitive interface for generating dedicated endpoints.
 * **Security**: Scaleway provides a secure environment for running your models. Our platform is built on top of a secure architecture, and we use state-of-the-art cloud security.
 * **Complete data privacy**: No storage or third-party access to your data (prompt or responses), to ensure it remains exclusively yours.
-* **Auto-scaling (coming soon)**: Scaleway LLM Inference automatically scales your instances based on demand, ensuring that your models are always available and responsive.
+* **Auto-scaling (coming soon)**: Scaleway Managed Inference automatically scales your instances based on demand, ensuring that your models are always available and responsive.
 
 <Message type="important">
   This service is in private beta. [Specific terms and conditions](https://www.scaleway.com/en/contracts/) apply.
 </Message>
```

Comment:
@tgenaitay shall we remove this?

Reply:
We can scratch "private" to anticipate a bit as public beta is set to start in July. But we're still beta phase, so specific terms still apply.

Reply:
```suggestion
<Message type="important">
  This service is in beta. [Specific terms and conditions](https://www.scaleway.com/en/contracts/) apply.
</Message>
```